btr-api/tests/unit/resources/test_submissions.py

```diff
@@ -13,6 +13,7 @@
 from sqlalchemy import text
```

Comment:
Can you add a test in here similar to 'test_post_plots' for the put? Ideally it would start with a realistic example of an SI and then the PUT would update an existing person with things like name, etc. and then also add a new person, and then it would verify the result is how we want the SI to look